insights/combiners/tests/test_sysctl_conf.py

```diff
@@ -0,0 +1,81 @@
+import doctest
+
+from insights.combiners import sysctl_conf
+from insights.parsers import sysctl
+from insights.tests import context_wrap
+
+
+SYSCTL_CONF_TEST1 = """
+# sysctl.conf sample
+#
+  kernel.domainname = example.com
+# kernel.domainname.invalid = notvalid.com
+
+; this one has a space which will be written to the sysctl!
+  kernel.modprobe = /sbin/mod probe
+""".strip()
+
+SYSCTL_CONF_TEST2 = """
+kernel.sysrq = 1
+kernel.panic_on_oops = 1
+kernel.panic_on_unrecovered_nmi = 1
+kernel.softlockup_panic = 1
+kernel.unknown_nmi_panic = 1
+kernel.nmi_watchdog = 1
+""".strip()
+
+SYSCTL_CONF_TEST3 = """
+vm.dirty_ratio = 15
+vm.dirty_background_ratio = 5
+vm.swappiness = 10
+"""
+
+SYSCTL_CONF_TEST4 = """
+vm.dirty_ratio = 30
+vm.dirty_background_ratio = 50
+vm.swappiness = 60
+kernel.sysrq = 16
+kernel.domainname = test.example.com
+"""
+
+SYSCTL1_CONF_FILENAME = "/etc/sysctl.d/10-test.conf"
+SYSCTL2_CONF_FILENAME = "/etc/sysctl.d/20-test.conf"
+SYSCTL3_CONF_FILENAME = "/etc/sysctl.d/30-test.conf"
+SYSCTL4_CONF_FILENAME = "/etc/sysctl.d/99-test.conf"
+
+
+def test_all_sysctl_conf():
+    conf_1 = sysctl.SysctlConf(context_wrap(SYSCTL_CONF_TEST1, path=SYSCTL1_CONF_FILENAME))
```

Comment:
It would be better that the path of `SysctlConf` should be `/etc/sysctl.conf`.

Reply:
Good catch I definitely missed that, I'll correct the paths based on the parser used.